label_studio/io_storages/s3/api.py

```diff
@@ -12,33 +14,108 @@ class S3ImportStorageListAPI(ImportStorageListAPI):
     queryset = S3ImportStorage.objects.all()
     serializer_class = S3ImportStorageSerializer
 
+    @swagger_auto_schema(
```

Comment:
@niklub Seems we should stop to write `@swagger_auto_schema` before all empty methods like get, post, ... and do it before class implementation, like here: 
https://github.com/heartexlabs/label-studio/blob/master/label_studio/data_manager/api.py#L53

Reply:
and what are the benefits?